src/auth/src/credentials.rs

```diff
@@ -165,3 +165,80 @@ pub mod traits {
         }
     }
 }
+
+/// The path to Application Default Credentials (ADC), as specified in [aip/4110].
+///
+/// [aip/4110]: https://google.aip.dev/auth/4110
+#[allow(dead_code)] // TODO(#442) - implementation in progress
+fn adc_path() -> String {
+    if let Ok(e) = std::env::var("GOOGLE_APPLICATION_CREDENTIALS") {
+        return e;
+    }
+    adc_well_known_path()
+}
+
+/// The well-known path to ADC on Windows, as specified in [aip/4113].
+///
+/// [aip/4113]: https://google.aip.dev/auth/4113
```

Comment:
Elsewhere we use `[AIP-4113]` for these links.  I think I like these better, but maybe we should stay consistent. Unless there is some precedent we should be following?

Reply:
Changed to `[AIP-XXXX]`